ports/glutin/window.rs

```diff
@@ -345,16 +345,16 @@ impl Window {
                     self.pending_key_event_char.set(Some(ch));
                 }
             }
-            Event::KeyboardInput(element_state, scan_code, Some(virtual_key_code)) => {
+            Event::KeyboardInput(element_state, scan_code, virtual_key_code) => {
                 match virtual_key_code {
-                    VirtualKeyCode::LControl => self.toggle_modifier(LEFT_CONTROL),
-                    VirtualKeyCode::RControl => self.toggle_modifier(RIGHT_CONTROL),
-                    VirtualKeyCode::LShift => self.toggle_modifier(LEFT_SHIFT),
-                    VirtualKeyCode::RShift => self.toggle_modifier(RIGHT_SHIFT),
-                    VirtualKeyCode::LAlt => self.toggle_modifier(LEFT_ALT),
-                    VirtualKeyCode::RAlt => self.toggle_modifier(RIGHT_ALT),
-                    VirtualKeyCode::LWin => self.toggle_modifier(LEFT_SUPER),
-                    VirtualKeyCode::RWin => self.toggle_modifier(RIGHT_SUPER),
+                    Some(VirtualKeyCode::LControl) => self.toggle_modifier(LEFT_CONTROL),
+                    Some(VirtualKeyCode::RControl) => self.toggle_modifier(RIGHT_CONTROL),
+                    Some(VirtualKeyCode::LShift) => self.toggle_modifier(LEFT_SHIFT),
+                    Some(VirtualKeyCode::RShift) => self.toggle_modifier(RIGHT_SHIFT),
+                    Some(VirtualKeyCode::LAlt) => self.toggle_modifier(LEFT_ALT),
+                    Some(VirtualKeyCode::RAlt) => self.toggle_modifier(RIGHT_ALT),
+                    Some(VirtualKeyCode::LWin) => self.toggle_modifier(LEFT_SUPER),
+                    Some(VirtualKeyCode::RWin) => self.toggle_modifier(RIGHT_SUPER),
```

Comment:
[This is the improvements](https://github.com/servo/servo/pull/15800/files#diff-f3afdd0d8092731ed5c423ce2e8dfd70L396) because now we are handling correctly the case where virtual key code is ```None```.